Sources/Danger/BitBucketServerDSL.swift

```diff
@@ -298,24 +290,43 @@ public struct BitBucketServerPR: Decodable, Equatable {
     public let isLocked: Bool
 
     /// The creator of the PR
-    public let author: BitBucketServerAuthor
+    public let author: Participant
 
     /// People requested as reviewers
-    public let reviewers: [BitBucketServerUser]
+    public let reviewers: [Reviewer]
 
     /// People who have participated in the PR
-    public let participants: [BitBucketServerAuthor]
+    public let participants: [Participant]
 
     // MARK: - BitBucketServerAuthor
 
-    public struct BitBucketServerAuthor: Decodable, Equatable {
+    /// A user that is parecipating in the PR
+    public struct Participant: Decodable, Equatable {
         enum CodingKeys: String, CodingKey {
             case user
         }
 
         /// The BitBucket Server User
         public let user: BitBucketServerUser
     }
+
+    /// A user that reviewed the PR
+    public struct Reviewer: Decodable, Equatable {
+        enum CodingKeys: String, CodingKey {
```

Comment:
hmm, why did you add `CodingKeys` enum? isn't that one synthesized for you already? 🤔